logging/fields/fields.go

```diff
@@ -220,15 +222,25 @@ func OperatorIDStr(operatorId string) zap.Field {
 func Height(height specqbft.Height) zap.Field {
 	return zap.Uint64(FieldHeight, uint64(height))
 }
+func GenesisHeight(height genesisspecqbft.Height) zap.Field {
+	return zap.Uint64(FieldHeight, uint64(height))
+}
 
 func Round(round specqbft.Round) zap.Field {
 	return zap.Uint64(FieldRound, uint64(round))
 }
+func GenesisRound(round genesisspecqbft.Round) zap.Field {
```

Comment:
probably Round will just work as the types are compatible and can be cast easily, by minimum move these to separate file.